web/react/components/more_direct_channels.jsx

```diff
@@ -47,6 +47,21 @@ export default class MoreDirectChannels extends React.Component {
         UserStore.addChangeListener(this.handleUserChange);
     }
 
+    componentDidUpdate(prevProps) {
+        if (!prevProps.show && this.props.show) {
+            this.onShow();
+        }
+    }
+
+    onShow() {
+        if ($(window).width() > 768) {
+            $(ReactDOM.findDOMNode(this.refs.modalBody)).perfectScrollbar();
```

Comment:
Is it intended that this now scrolls the search box in the modal? The reason it used `this.refs.userList` instead of `this.refs.modalBody` before was to only scroll the user list itself.